opencti-platform/opencti-graphql/src/schema/stixRefRelationship.ts

```diff
@@ -727,7 +726,9 @@ export const objectOrganization: RefAttribute = {
   multiple: true,
   upsert: true,
   isRefExistingForTypes(this, fromType, toType) {
-    return !(fromType === ENTITY_TYPE_EVENT || isStixDomainObjectIdentity(fromType)
+    return !(fromType === ENTITY_TYPE_EVENT
+        || fromType === ENTITY_TYPE_IDENTITY_ORGANIZATION
+        || fromType === ENTITY_TYPE_IDENTITY_SECTOR
```

Comment:
Not sure to understand why we replace the call to identity function and replace it with 2 conditions ?

Reply:
`Organization` and `Sector`  are not concerned by organization segregation, we need to test for these specific Entites to exclude them from sharing

Reply:
But was `isStixDomainObjectIdentity(fromType)` not already doing it ?

Reply:
Ah, but System and and Individual are also Indenties but for them we want the sharing to be working so the function was to restrictive ?

Reply:
Had the answer in the description oft he PR